### DISCUSSION TITLE
Undefined type 'Livewire\Component'.

### DISCUSSION THREAD

**Discussion by RaulAcevedoF** [livewire/livewire#4940] [1. Help] [answered]
I recently started working with Livewire . Creating my first component I got an error when my class extends from Component (Undefined type 'Livewire\Component'.).

I followed the steps indicated in the documentation to be able to use Livewire and I can not solve the error. 

Could someone please guide me thank you !!!
![livewire](https://user-images.githubusercontent.com/51192032/166847951-2bc7fbd7-6f8f-43e7-b918-9f6945e677de.png)

**Comment by AlexisRamirezHernandez** [comment]
I have same problem, do you know the solution for it?

**Comment by AlexisRamirezHernandez** [comment, marked as answer]
Oh, I can resolve this problem, you entry in your console the next commands:
`composer dump-autoload
php artisan package:discover 
php artisan vendor:publish --tag=livewire:config
composer require livewire/livewire`
Aditional: you should rename your blade template as, app.blade.php

**Reply by AlexisRamirezHernandez** [reply]
Ah and restart your IDE or editor

**Reply by RaulAcevedoF** [reply]
Thanks @AlexisRamirezHernandez Worked excellent!
